hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -0,0 +1,1584 @@
+          schema:
+            type: integer
+        - name: prevKey
+          in: query
+          description: Only return keys that are present after the given key prevKey prefix
+          required: false
+          schema:
+            type: string
+            example: /vol1/bucket1/key1
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/KeyMetadata'
+  /containers/missing:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get the MissingContainerMetadata for all missing containers
+      operationId: getMissingContainers
+      parameters:
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MissingContainerMetadata'
+  /containers/{containerId}/replicaHistory:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get all the Replica history for a given container id
+      operationId: getReplicaHistory
+      parameters:
+        - name: containerId
+          in: path
+          description: ID of the container for which we want Key Metadata
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ReplicaHistory'
+  /containers/unhealthy:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get UnhealthyContainerMetadata for all the unhealthy containers 
+      operationId: getUnhealthyContainers
+      parameters:
+        - name: batchNum
+          in: query
+          description: Size of the batch for the result. It will give us results from **(limit + 1) to (2 * limit)**
+          required: false
+          schema:
+            type: integer
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnhealthyContainerMetadata'
+  /containers/unhealthy/{containerState}:
```

Comment:
check the param `containerState`, should be `state`

Reply:
Sure, this is the same as https://github.com/apache/ozone/pull/5509#discussion_r1381910275 but we can change it

Reply:
> Sure, this is the same as [#5509 (comment)](https://github.com/apache/ozone/pull/5509#discussion_r1381910275) but we can change it

Inline with code looks uniform as Swagger yaml being open standard , so not to just think about link, I feel we should have it in line with code along with param description.

Reply:
Addressed in the latest commit, reverted to {state}